ci-operator/config/openshift/cluster-operator/master.json

```diff
@@ -0,0 +1,89 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "root": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "release",
+      "tag": "golang-1.10"
+    },
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    },
+    "ansible": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "ansible"
+    },
+    "test_base_image": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "release",
+      "tag": "golang-1.10"
+    }
+  },
+  "canonical_go_repository": "github.com/openshift/cluster-operator",
```

Comment:
Do we need this if it's not anything special? @smarterclayton

Reply:
We should keep it in case we fork or move it, and this way it also works if you run the operator directly against your fork branch.